src/gradualizer_type.erl

```diff
@@ -21,7 +21,7 @@
 -export_type([abstract_type/0]).
 
 %% Export the additional types that gradualizer uses
--export_type([af_character/0]).
+-export_type([af_pattern/0]).
```

Comment:
How about naming the exported type `abstract_pattern`, similarly to the other exported types?